ansible_collections/arista/avd/molecule/evpn_underlay_isis_overlay_ibgp/documentation/devices/DC1-BL1A.md

```diff
@@ -415,10 +420,10 @@ interface Port-Channel5
 
 ##### ISIS
 
-| Interface | ISIS instance | ISIS metric | Interface mode |
-| --------- | ------------- | ----------- | -------------- |
-| Loopback0 | EVPN_UNDERLAY | - | passive |
-| Loopback1 | EVPN_UNDERLAY | - | passive |
+| Interface | ISIS instance | ISIS BFD | ISIS metric | Interface mode |
```

Comment:
Please remove the BFD column from loopbacks.

Reply:
but in eos_cli_config_gen we have implemented BFD for loopbacks also.

Reply:
removed it.